app/views/proposal_mailer/proposal_complete.html.haml

```diff
@@ -1,15 +1,45 @@
-%table.full-width.header
-  %tr
-    %td.hero
-    %td
-      = image_tag "emails/icon-thumbs-up-circle.png", class: "header-icon"
-
-%table.two-thirds-width
-  %tr
-    %td.header
-      = t("mailer.proposal_mailer.proposal_completed.header")
-
-%table.two-thirds-width
-  %tr
-    %td.subheader
-      = t("mailer.proposal_mailer.proposal_completed.subheader")
+- content_for :header_icon, "emails/icon-thumbs-up-circle.png"
+- top_head = t("mailer.proposal_mailer.proposal_completed.header")
```

Comment:
since this email is called `proposal_complete`, let's make sure the locale keys match - rename to `proposal_complete` in there and update these?